client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,5 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
```

Comment:
The width is messed up if we don't have this. I could figure it out properly, but ... lazy lazy